core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -688,6 +692,11 @@ public static String reverse(String s) {
     return buf.reverse().toString();
   }
 
+  /** SQL "LEVENSHTEIN(string1, string2)" function. */
```

Comment:
In order to consistent programming style, Please remove double quotation marks.

Reply:
Thanks for the review, I will fix it soon.

Reply:
Fixed.